community/consistency-check/src/test/java/org/neo4j/consistency/ConsistencyCheckServiceIntegrationTest.java

```diff
@@ -89,7 +89,7 @@ protected void generateInitialData( GraphDatabaseService graphDb )
         }
     };
 
-    private final TestDirectory testDirectory = TestDirectory.testDirectory();
+    private final TestDirectory testDirectory = TestDirectory.testDirectory( ConsistencyCheckServiceIntegrationTest.class );
```

Comment:
Why do you need to pass class explicitly in all those tests? TestDirectory rule will gonna get it from description object that JUnit runner will supply?

Reply:
Oh, right. It's a completely different reason that collusions on directories were possible.